weasyl/test/searchtag/test_edit_global_tag_restrictions.py

```diff
@@ -22,7 +22,7 @@ def test_edit_global_tag_restrictions_fully_clear_entries_after_adding_items(mon
     searchtag.edit_global_tag_restrictions(director_user_id, tags)
     tags = set()
     searchtag.edit_global_tag_restrictions(director_user_id, tags)
-    assert searchtag.get_global_tag_restrictions(director_user_id) == []
+    assert not searchtag.get_global_tag_restrictions(director_user_id)
```

Comment:
Nit: I prefer `== {}`  because of the possible weird case where get_global_tag_restrictions returns `None` due to a bug.